.github/ISSUE_TEMPLATE/questions.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Questions are usually asked in Slack or Discuss. Example in the istio/istio template: `To ask questions about how to use Istio, please visit https://discuss.istio.io/`.